src/modules/admin/wrappers/with-admin-core-config.tsx

```diff
@@ -104,21 +104,25 @@ export const withAdminCoreConfig = (WrappedComponent: ComponentType): ComponentT
 					componentProps: {
 						add: { name: 'plus' },
 						view: { name: 'show' },
-						angleDown: { name: 'angle-down' },
-						angleUp: { name: 'angle-up' },
+						angleDown: { name: 'angle-down--light' },
```

Comment:
Q: Can we use the `IconNamesLight` to add the icons? 
```suggestion
						angleDown: { name: IconNamesLight.AngleDown },
```